doc/releases/migration-guide-3.6.rst

```diff
@@ -430,6 +430,32 @@ Networking
   cannot be accessed from usermode thread. This means that the API calls will need to made
   from supervisor mode thread.
 
+zcbor
+=====
+
+* If you have zcbor-generated code that relies on the zcbor libraries through Zephyr, you must
+  regenerate the files using zcbor 0.8.0. Note that the names of generated types and members has
+  been overhauled, so the code using the generated code must likely be changed.
+  For example:
+    * Leading single underscores and all double underscores are largely gone,
+    * Names sometimes gain suffixes like ``_m`` or ``_l`` for disambiguation.
+    * All enum (choice) names have now gained a ``_c`` suffix, so the enum name no longer matches
+      the corresponding member name exactly (because this broke C++ namespace rules).
```

Comment:
probably should be: 

```suggestion
  For example:
  
  * Leading single underscores and all double underscores are largely gone,
  * Names sometimes gain suffixes like ``_m`` or ``_l`` for disambiguation.
  * All enum (choice) names have now gained a ``_c`` suffix, so the enum name no longer matches
      the corresponding member name exactly (because this broke C++ namespace rules).
```